src/shell.ml

```diff
@@ -39,7 +39,7 @@ let run args target =
   let cmd = String.concat " " (List.map quote_filename_if_needed args) in
   if !*My_unix.is_degraded || Sys.os_type = "Win32" then
     begin
-      Log.event cmd target Tags.empty;
+      if false then Log.event cmd target Tags.empty;
```

Comment:
Why is this disabled?
```suggestion
      Log.event cmd target Tags.empty;
```

Reply:
It make the testsuite fail because it output `cmd` on stdout.
I wonder if we should prefix the command with `"+ "` similar to what is done in non-degraded mode. The testsuite filters theses lines out already.

Reply:
I've added the `+ ` prefix